Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```

Comment:
i thought this was just removing the `svc` ??

Reply:
like: `registry.ci.openshift.org/openshift/release:golang-1.15`

Reply:
This just matches https://github.com/openshift/machine-config-operator/blob/master/Dockerfile.rhel7#L2

The previous edition of it rather: https://github.com/openshift/machine-config-operator/pull/2337/files

Maybe we shouldn't reference builder? I'm not sure. I just know the updated location does work. `registry.ci.openshift.org/openshift/release:golang-1.15` does not seem to exist.